terraform/projects/app-ecs-services/alertmanager-service.tf

```diff
@@ -154,6 +154,10 @@ data "pass_password" "dgu_pagerduty_service_key" {
   path = "pagerduty/integration-keys/dgu"
 }
 
+data "pass_password" "registers_zendesk" {
```

Comment:
Need to add this data into the template file for `alertmanager_config_file`

Reply:
Well spotted! Have fixed it now.